FrontEnd/scripts/works.js

```diff
@@ -261,9 +313,9 @@ addWorkForm.addEventListener("submit", async (event) => {
   }
 
   const formData = new FormData(addWorkForm);
-
   const newWorkTitle = formData.get("title");
   const newWorkCategory = formData.get("category");
+  const newWorkImage = formData.get("image");
```

Comment:
I don't think you should perform validation here, as you can access isTitleValid and siblings ...